k8s/components/gridsuite/patches/cleanup-voltageinit-sidecar.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: not-important
+spec:
+  template:
+    spec:
+      containers:
+        - name: cleanup
+          image: busybox:1.36.1
+          volumeMounts:
+            - mountPath: /tmp
+              name: tmp-emptydir
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            allowPrivilegeEscalation: false
+            runAsNonRoot: true
+            readOnlyRootFilesystem: true
+          command:
+            - "/bin/sh"
+            - "-c"
+            - |
+              while true; do
+                total_size=$(du -s /tmp | awk '{print $1}')
+                if [ $total_size -gt 80000 ]; then
+                  echo "$(date -u +"%Y-%m-%dT%H:%M:%SZ") - Cleaning up /tmp..."
+                  ls -d -t /tmp/*/ | tail -n +6 | xargs rm -rf
```

Comment:
This doesn't seem to exclude any particular directories temp directory (e.g. /tmp/hsperfdata_powsybl or /tmp/tomcat or /tmp/itools...). Should these directories be deleted as well ? or should they stay available for the lifetime of the pod ?